tests/Install-Scripts.Test/GivenThatIWantToInstallDotnetFromAScript.cs

```diff
@@ -248,9 +252,7 @@ public void WhenInstallingAspNetCoreRuntime(string channel, string? quality, str
         {
             if (channel == "release/3.0"
                 || channel == "release/3.1"
-                || channel == "release/5.0"
-                ||
-                channel == "5.0" && !RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                || channel == "release/5.0")
```

Comment:
I would prefer we comment out lines in list above - it makes more obvious how many cases are disabled.

Reply:
Makes sense, will do 👍